### PR TITLE
306948: Added PE vars for flux manifest infra patch

### PR DESCRIPTION
**Pull request by jeevankuduvaravindran**
# **What this PR does / why we need it**:
This PR contains the private endpoint variables to flux manifest patch files
[AB#306948](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/306948)

# **Special notes for your reviewer**
These files were generated from flux manifest automation from adp-infrastructure-core.
Part of the PR https://github.com/DEFRA/adp-infrastructure-core/pull/128 

# Testing
*Any relevant testing information and pipeline runs.*

# Checklist (please delete before completing or setting auto-complete)
- [x] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [x] Title pattern should be `{work item number}: {title}`
- [x] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

# **How does this PR make you feel**:
gif

### CODE REVIEW

**Review by ganeshg2491 — APPROVED** [review]

File: services/ffc/ffc-demo/ffc-demo-payment-service/infra/snd/01/patch.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ spec:
   chart:
     spec:
       chart: ffc-demo-payment-service-infra
-      version: "4.2.24" # {"$imagepolicy": "flux-config:ffc-demo-payment-service-snd-01:tag"}
+      version: "0.1.0" # {"$imagepolicy": "flux-config:ffc-demo-payment-service-snd-01:tag"}
```
Comment (DaRoszko) [inline services/ffc/ffc-demo/ffc-demo-payment-service/infra/snd/01/patch.yaml:10]: all of these shouldnt be reverted?

**Review by kbabigumira — CHANGES_REQUESTED** [review]
for the ffc-dal service, we should add all the environments (snd3, dev1,...., prd1)

File: services/ffc/ffc-demo/ffc-demo-payment-service/infra/snd/01/patch.yaml (hunk 1 above)
Comment (kbabigumira) [reply services/ffc/ffc-demo/ffc-demo-payment-service/infra/snd/01/patch.yaml:10]: @DaRoszko  yes. We discussed this with @aadev1  and @jeevankuduvaravindran . Asad is updating the script to do incremental updates, leaving current version as is if greater than 0.1.0

File: services/ffc/ffc-dal/fcp-data-access-layer-api/infra/snd/03/patch.yaml [hunk 2]
```diff
@@ -7,21 +7,27 @@ spec:
...
       chart: fcp-data-access-layer-api-infra
-      version: "1.0.34" # {"$imagepolicy": "flux-config:fcp-data-access-layer-api-snd-03:tag"}
+      version: "0.1.0" # {"$imagepolicy": "flux-config:fcp-data-access-layer-api-snd-03:tag"}
   values:
     namespace: ${TEAM_NAMESPACE}
     fluxConfigNamespace: ${FLUX_CONFIG_NAMESPACE}
     subscriptionId: ${SUBSCRIPTION_ID}
     serviceBusResourceGroupName: ${SERVICES_INFRA_RG}
     serviceBusNamespaceName: ${SHARED_SERVICEBUS_NAME}
-    postgresResourceGroupName: ${DATABASE_SERVER_RG}
```
[the remark this answers is not in the document]
Comment (jeevankuduvaravindran) [reply services/ffc/ffc-dal/fcp-data-access-layer-api/infra/snd/03/patch.yaml:17]: @kbabigumira This is bcoz when generating it from automation, it adds the variables conditionally if the backend has database

**Pull request closed without merging**